docs/app/globals.css

```diff
@@ -2,8 +2,8 @@
 
 @layer base {
   @import "./syntax-highlighting.css";
-  @import "./fonts/cmun/cmun-serif.css";
-  @import "mafs/index.css";
+  @import "../../core.css";
+  @import "../../font.css";
```

Comment:
Unfortunately, Next's compiler setup didn't understand `mafs/../core.css` so this is a little wonky.